SECURITY.md

```diff
@@ -0,0 +1,37 @@
+# oci-spec-rs Security
+
+Security is taken seriously and has high priority across all related projects to
+ensure users can trust this project for their systems.
+
+We're extremely grateful for security researchers and users that report
+vulnerabilities to the community. All reports are thoroughly investigated by a
+set of community volunteers.
+
+## Report a Vulnerability
+
+<!-- TODO: the mailing list has to be requested -->
+
+To make a report, email the vulnerability to the private
+[cncf-oci-spec-rs-security@lists.cncf.io](mailto:cncf-crio-security@lists.cncf.io) list
```

Comment:
I assume that these two addresses not matching is linked to the above TODO, right?